src/styles/_variables.scss

```
@@ -1,3 +1,14 @@
+$cardDefaultSpacing: 10px;
+$cardDefaultBorderRadius: 8px;
+$cardDefaultHoverBgColor: #ffffff;
+$cardDefaultBorderWidth: 1px;
+$cardDefaultDisabledOpacity: 0.4;
+$cardTagDefaultBorderRadius: 2px;
+
+$cardTypes: 'success' 'green', 'informational' 'blue', 'error' 'red',
+  'attention' 'brown', 'warning' 'orange', 'neutral' 'gray';
```

```suggestion
$cardTypes: (
    'success': 'green',
    'informational': 'blue',
    'error': 'red',
    'attention': 'brown',
    'warning': 'orange',
    'neutral': 'gray',
);
```
IMO taki zapis listy sass-owej jest znacznie czytelniejszy.